websites/0-9/20XX/dist/metadata.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "Snazzah",
+    "id": "158049329150427136"
+  },
+  "url": "20xx.io",
+  "description": {
+    "en": "20XX is a unique platform MOBA which includes a variety of characters and maps to play on."
+  },
+  "service": "20XX",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/UkDzKKL.png",
```

Comment:
Can you change the logo to a better quality version?

Reply:
If you can't find one, just use something like [waifu2x](http://waifu2x.udp.jp/)

Reply:
Not too sure, that's the best quality I can really get since it came from an waifu2x upscaled version of the site's logo (which is 64x64)

Reply:
I can possibly use [this image](https://20xx.io/nxc/img/aes/unl/CHAR_BOX.png) instead. It won't have the colors and such, but it has the icon shape and the correct dimensions.